test/src/test/java/aima/test/unit/probability/BigDecimalTypeTest.java

```diff
@@ -0,0 +1,112 @@
+package aima.test.unit.probability;
+
+import aima.extra.probability.*;
+import static org.junit.Assert.*;
+import java.math.*;
+
+import org.junit.Test;
+
+public class BigDecimalTypeTest {
+
```

Comment:
NOTE: no tests should output to system.out (i.e. if all tests successfully no output should go to the console).
Also, it would be nice to see a common set of probability tests that work irrespective of the underlying implementation (like some of the parameterized search tests, e.g.: UniformCostSearchTest.